plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -341,6 +342,9 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
       if( options.count( "chain-state-db-size-mb" ))
          my->chain_config->state_size = options.at( "chain-state-db-size-mb" ).as<uint64_t>() * 1024 * 1024;
 
+      if( options.count( "chain-state-db-guard-size-mb" ))
```

Comment:
Forget this comment, missed it was a **minimum** size

Reply:
although this is fair, it looks like I was intending the track to be 512k... hrm